src/models/Tasks.js

```diff
@@ -1,23 +1,23 @@
-import mongoose from 'mongoose';
+import mongoose from "mongoose";
 // const mongoose = require("mongoose")
 
 const TaskSchema = mongoose.Schema({
-    name: {
-        type: String,
-        required: true,
-    },
-    createdby: {
-        type: String,
-        require: false,
-    },
-    description: {
-        type: String,
-        required: false,
-    },
-    points: {
-        type: String,
-        required: false,
-    },
-})
+  name: {
+    type: String,
+    required: true,
+  },
+  createdBy: {
```

Comment:
- updated here
- Swagger documentation will still need to be updated
- @jfarelli if you're calling `.createdby` anywhere in the FE, it'll need to be updated to `.createdBy`